app/src/main/res/values/static_strings.xml

```diff
@@ -34,6 +34,8 @@
     <string name="preferences_debug_settings">Secret Settings</string>
     <!-- Label for the secret settings preference -->
     <string name="preferences_debug_info" translatable="false">Secret Debug Info</string>
+    <!-- Label for the Nimbus experiments preference -->
+    <string name="preferences_nimbus_experiments">Nimbus Experiments</string>
```

Comment:
This could be moved to AC 📦

Reply:
I chose to keep this here since it's a static string (meaning it won't be translated) since this is only an internally used string.